client.go

```diff
@@ -88,6 +87,7 @@ func (lc *StreamClient) Publish(ctx context.Context, payload io.Reader, key io.R
 	ce.Source = "source-todo" // TODO
 	ce.Type = "riff-event"    // TODO
 	ce.Id = uuid.New().String()
+	ce.Time = time.Now().String()
```

Comment:
this is a good default, but we'll eventually want users to be able to define the value along side source and event